packages/desktop-client/src/components/table.tsx

```diff
@@ -261,7 +265,21 @@ export function Cell({
               textOverflow: 'ellipsis',
             }}
           >
-            {formatter ? formatter(value) : value}
+            {icon ? (
+              <Button bare>
```

Comment:
Very small UI nits:

* the ones with a button are a slightly different appearance than the ones without (they appear to be a darker color & have font weight 400 instead of 300), it would be nice to keep them looking the same as now
* it feels a little weird to me that clicking on “Payment” works but doesn’t add an icon there… it would be nice to have the button background extend to cover both the Payment and Deposit amounts (or something?). Sounds challenging, feel free to punt this detail to an issue/future PR if you can’t come up with a way to do something similar
* The natural padding of the `Button` brings the start of the heading title slightly to the right of the text inside the table. The plain text labels look to be about 1px too far to the left on `master`, so it would be nice to tighten those up along with adding a negative left margin for the button to make sure all columns line up well.

Reply:
- Changed button color and weight to match existing table header so they don't look different from `master`.
- I gave payment and deposit their own icons. They effectively do the same thing but have different default behavior. This does feel like a better UI, thanks!
- negative left margin cuts off the button. Not sure the best option as they don't line up in master...(have a look at balance column in master)

Reply:
> negative left margin cuts off the button

It looks like one of the parent elements (the direct parent of the `<button>` in the HTML) has `overflow: hidden` which is causing this. Might be best to switch that to `overflow: visible`, at least in the X direction? Although I’m not sure why that is there, maybe there’s a good reason

> Not sure the best option as they don't line up in master

I think payment/deposit are right-aligned, the others are left-aligned?